feat(changelog): jobs-added-custom-ephemeral-storage-on-serverless-jobs 2024-03-20 (#2926)
scaleway/docs-content: changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx

---
title: Custom Ephemeral Storage on Serverless Jobs
status: added
author:
    fullname: 'Join the #serverless-jobs-beta channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-03-20
category: serverless
product: jobs
---

> Serverless Jobs now offers the possibility to customise your custom ephemeral storage! 

Reviewer: ```suggestion
Serverless Jobs now offers the possibility of applying customized ephemeral storage! 
```

Reviewer 2: ```suggestion
Serverless Jobs now offers the possibility to customise you ephemeral storage! 
```
cc @nerda-codes what do you think?

> Each Serverless Job has a limited storage a now you can customise it to allow more demanding use cases.

Reviewer: ```suggestion
With custom ephemeral storage, your Serverless Jobs no longer have a limited storage, as you can customize them to allow more demanding use cases.
```

Reviewer 2: Ephemeral storage is still limited to 10GB (used to be 5), so maybe
```suggestion
Each Serverless Job comes with an ephemeral storage capacity that you can now customize to accommodate for more demanding use cases.
```

Reviewer 2: cc @nerda-codes

> Each Serverless Job comes with an ephemeral storage capacity that you can now customize to accommodate for more demanding use cases.

Reviewer 3: ```suggestion
Each Serverless Job comes with an ephemeral storage capacity that you can now customize to accommodate more demanding use cases.
```
